src/main/java/net/minestom/server/item/meta/EnchantedBookMeta.java

```diff
@@ -0,0 +1,67 @@
+package net.minestom.server.item.meta;
+
+import net.minestom.server.item.Enchantment;
+import net.minestom.server.item.ItemMeta;
+import net.minestom.server.item.ItemMetaBuilder;
+import net.minestom.server.utils.NBTUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jglrxavpok.hephaistos.nbt.NBTCompound;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class EnchantedBookMeta extends ItemMeta implements ItemMetaBuilder.Provider<EnchantedBookMeta.Builder> {
+
+    private final Map<Enchantment, Short> storedEnchantmentMap;
+
+    protected EnchantedBookMeta(@NotNull ItemMetaBuilder metaBuilder, Map<Enchantment, Short> storedEnchantmentMap) {
+        super(metaBuilder);
+        this.storedEnchantmentMap = new HashMap<>(storedEnchantmentMap);
+    }
+
+    /**
+     * Gets the stored enchantment map.
+     * Stored enchantments are used on enchanted book.
+     *
+     * @return an unmodifiable map containing the item stored enchantments
+     */
+    public @NotNull Map<Enchantment, Short> getStoredEnchantmentMap() {
+        return Collections.unmodifiableMap(storedEnchantmentMap);
+    }
+
+    public static class Builder extends ItemMetaBuilder {
+
+        private Map<Enchantment, Short> enchantments = new HashMap<>();
```

Comment:
Should we have a getter for this map to retrieve currently stored enchantments?

Reply:
You mean in the builder? I dont think that we should have any getter there